File: go/leetcode-26.go
package main

import "fmt"

func moveZeroes(nums []int)  {
	var n int
	for i := 0; i < len(nums); i++ {
		if nums[i] != 0{
			nums[n] = nums[i]
			n++
		}

	}
	for{
		if n < len(nums){
			nums[n] = 0
			n++
		}
		break
	}
}
func main() {
	nums := []int{0,1,0,3,12}
	moveZeroes(nums)
	fmt.Println(nums)
}

File: python3/leetcode-26.py
# python3
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        n = 0
        for i in range(len(nums)):
            if i == 0:
                nums[n] = nums[i]
                n = n + 1
            if i != 0 and nums[i] != nums[i - 1]:
                nums[n] = nums[i]
                n = n + 1
        return n